test/e2e_federation/upgrade.go

```diff
@@ -25,9 +25,7 @@ import (
 	. "github.com/onsi/ginkgo"
 )
 
-var upgradeTests = []upgrades.Test{
-	upgrades.SecretUpgradeTest(),
-}
+var upgradeTests = upgrades.GenericUpgradeTests()
```

Comment:
I'm not sure how I'm feeling about the "Generic" name, but I can't think of a more descriptive moniker offhand.

Could we just remove "Generic" altogether and make this UpgradeTests(), or just Tests()? I'm not sure that Generic is adding any semantic information.